schedule/yast/remote_ssh/remote_ssh_controller_sle15.yaml

```diff
@@ -11,7 +11,8 @@ schedule:
   - remote/remote_controller
   - installation/welcome
   - installation/accept_license
-  - installation/scc_registration
+  - installation/registration/register_via_scc
+  - installation/registration/modules_selection
```

Comment:
No libyui here

Reply:
we are aware, I guess it was too soon to create the ticket, because the path to the files will change due to  I also suggested Alex when grouping openSUSE yaml files to do the same for these offline/MM,etc., so they will be easy to spot, but still it is not done.